lib/rubocop/cop/style/documentation.rb

```diff
@@ -47,18 +47,25 @@ def check(ast, ast_with_comments)
           end
         end
 
+        def module_definition?(node)
+          # 'Const = Class.new' or 'Const = Module.new' are module definitions
+          node.class_type? ||
+            node.module_type? ||
+            (node.casgn_type? &&
+              (rhs = node.children[2]).send_type? &&
```

Comment:
I'll try to find a way to make the code read better. Regarding `module_definition?`, I chose that name because (technically) Ruby classes are also modules, though many programmers may think of them as distinct.